src/lib/SCREEN/menu.cpp

```diff
@@ -631,17 +635,32 @@ fsm_state_entry_t const main_menu_fsm[] = {
     {STATE_LAST}
 };
 
+// Linkstats FSM
+fsm_state_event_t const linkstats_confirm_events[] = {
+    {EVENT_TIMEOUT, GOTO(STATE_LINKSTATS)},
+    {EVENT_LONG_ENTER, PUSH(main_menu_fsm)},
+    {EVENT_LONG_RIGHT, PUSH(main_menu_fsm)},
+    {EVENT_UP, ACTION_POPALL},
+    {EVENT_DOWN, ACTION_POPALL}
+};
+fsm_state_entry_t const linkstats_menu_fsm[] = {
+    {STATE_LINKSTATS, nullptr, displayLinkstats, 1000, linkstats_confirm_events, ARRAY_SIZE(linkstats_confirm_events)},
+    {STATE_LAST}
+};
+
 // Entry FSM
 fsm_state_event_t const splash_events[] = {
     {EVENT_TIMEOUT, GOTO(STATE_IDLE)}
 };
 fsm_state_event_t const idle_events[] = {
     {EVENT_TIMEOUT, GOTO(STATE_IDLE)},
     {EVENT_LONG_ENTER, PUSH(main_menu_fsm)},
-    {EVENT_LONG_RIGHT, PUSH(main_menu_fsm)}
+    {EVENT_LONG_RIGHT, PUSH(main_menu_fsm)},
+    {EVENT_UP, PUSH(linkstats_menu_fsm)},
+    {EVENT_DOWN, PUSH(linkstats_menu_fsm)}
 };
 fsm_state_entry_t const entry_fsm[] = {
-    {STATE_SPLASH, nullptr, displaySplashScreen, 5000, splash_events, ARRAY_SIZE(splash_events)},
+    {STATE_SPLASH, nullptr, displaySplashScreen, 1000, splash_events, ARRAY_SIZE(splash_events)},
```

Comment:
Did you really mean to shorten this to 1 second? Or was it just for debugging?

Reply:
Debugging... but it would be nice to make it a little shorter. 3000?

Reply:
I'm cool with that.